operator/main.go

```diff
@@ -124,6 +119,17 @@ func main() {
 	setupManager(flagVar, controllers.NewCacheFunc(), scheme)
 }
 
+func configLogger() logr.Logger {
```

Comment:
Can we verify what the correct logger settings are for CLS / Logging Service with Monitoring Team? I think we should then use these settings and apply them globally

Reply:
Yes, we have one https://github.com/kyma-project/community/tree/main/concepts/observability-consistent-logging, I'll have a check and do changes accordingly.

Reply:
with currently refactoring, one example log like below:
```
{
    "level": "INFO",
    "date": "2022-11-03T15:01:27.294739+01:00",
    "caller": "controllers/kyma_controller.go:80",
    "msg": "reconciling modules",
    "context": {
        "controller": "kyma",
        "controllerGroup": "operator.kyma-project.io",
        "controllerKind": "Kyma",
        "Kyma": {
            "name": "kyma-xin-1-0",
            "namespace": "default"
        },
        "namespace": "default",
        "name": "kyma-xin-1-0",
        "reconcileID": "5322de8a-ebfa-4c3a-b5c0-2ac751f80165"
    }
}
```